gravitee-management-api-service/src/main/java/io/gravitee/management/service/impl/ApiServiceImpl.java

```diff
@@ -556,8 +564,30 @@ public String convertAsJsonForExport(final String apiId) {
         apiEntity.setDeployedAt(null);
         apiEntity.setPrimaryOwner(null);
         apiEntity.setState(null);
+
```

Comment:
About your stacktrace when you try to import your api definition.

This is because the branch you fetch doesn't exists anymore.

if you replace `"branchOrTag": "feat/first.version",`to `"branchOrTag": "master",` it works